portal/src/components/Header/components/FullScreenMenu.tsx

```diff
@@ -86,7 +86,7 @@ export const FullScreenMenu: React.FC<FullScreenMenuProps> = ({
     return (
         <>
             {CustomButton ? (
-                <CustomButton />
+                <CustomButton active={isActive} />
```

Comment:
```suggestion
                <CustomButton active={isOpen} />
```

Begge deler funker, men `isOpen` er hakket riktigere 😊